src/util.js

```diff
@@ -22,6 +22,22 @@
 const path = require('path');
 const child_process = require("child_process");
 
+/**
+ * Form CLI arguments into a single string.
+ * @param { Array } argv - Argument vector.
+ */
+function cli_args(argv) {
+  let result = '';
+  argv.forEach(function (element) {
+    // XXX: We wrap double quotes if the string contains spaces
+    if (/\s/g.test(element)) {
```

Comment:
Got it from, https://stackoverflow.com/questions/1731190/check-if-a-string-has-white-space